compose/compose.go

```diff
@@ -752,6 +844,26 @@ func main() {
 		}
 	}
 
+	if opts.PDir != "" {
+		if _, err := os.Stat(opts.DataDir); !os.IsNotExist(err) {
+			fatal(errors.Errorf("Directory %v already exists.", opts.DataDir))
+		}
+
+		n := 1
+		for n <= opts.NumAlphas {
+			new_dir := opts.DataDir + "/alpha" + strconv.Itoa(n) + "/p"
```

Comment:
nitpick: `new_dir` doesn't conform to mixedCaps or the naming convention in the above nitpick -- best to keep the style consistent

Reply:
Done.